From: Danny Kukawka <danny.kukawka@web.de>
Subject: fixed merge for uint64

Fixed typo to get merge working again with type=uint64. 

Should fix fd.o bug #16575. 

Signed-off-by: Danny Kukawka <danny.kukawka@web.de>
---

 create_cache.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hald/create_cache.c b/hald/create_cache.c
index cdc8c9a..d52f834 100644
--- a/hald/create_cache.c
+++ b/hald/create_cache.c
@@ -159,7 +159,7 @@ get_merge_type (const char *str)
 		return MERGE_BOOLEAN;
 	if (strcmp (str, "int") == 0)
 		return MERGE_INT32;
-	if (strcmp (str, "unint64") == 0)
+	if (strcmp (str, "uint64") == 0)
 		return MERGE_UINT64;
 	if (strcmp (str, "double") == 0)
 		return MERGE_DOUBLE;
